moonshot/integrations/web_api/routes/cookbook.py

```diff
@@ -59,23 +60,23 @@ def get_all_cookbooks(
     ),
     count: bool = Query(False, description="Whether to include the count of recipes"),
     cookbook_service: CookbookService = Depends(Provide[Container.cookbook_service]),
-) -> list:
+) -> list[CookbookResponseModel]:
```

Comment:
@imda-normanchia 
was this updated because of my suggestion during our discussion yesterday?

Reply:
commented the wrong line